config/fcos/v1_6_exp/validate_test.go

```diff
@@ -237,6 +237,134 @@ func TestValidateGrubUser(t *testing.T) {
 	}
 }
 
+func TestValidateMountPoints(t *testing.T) {
+	tests := []struct {
+		in      Config
+		out     error
+		errPath path.ContextPath
+	}{
+		// valid config (has prefix "/etc" or "/var")
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path:          util.StrToPtr("/etc/foo"),
+								WithMountUnit: util.BoolToPtr(false),
+							},
+							{
+								Path:          util.StrToPtr("/var"),
+								WithMountUnit: util.BoolToPtr(false),
+							},
+							{
+								Path:          util.StrToPtr("invalid/path"),
+								WithMountUnit: util.BoolToPtr(true),
+							},
+							{
+								WithMountUnit: util.BoolToPtr(false),
+							},
+						},
+					},
+				},
+			},
+			out:     common.ErrMountPointForbidden,
+			errPath: path.New("yaml", "storage", "filesystems", 2, "path"),
+		},
+		// invalid config (path name is '/')
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: util.StrToPtr("/"),
+							},
+						},
+					},
+				},
+			},
+		},
+		// valid config (path is nil)
+		{
+			in: Config{
+				Config: base.Config{
+					Storage: base.Storage{
+						Filesystems: []base.Filesystem{
+							{
+								Path: nil,
+							},
```

Comment:
Missing `WithMountUnit: true`.

Reply:
Have dropped the test

Reply:
Have aded Withmountunit and moved to the first test 

		// valid config (has prefix "/etc" or "/var")